js/interactive-guides/microprofile-config/microprofile-config-callback.js

```diff
@@ -196,53 +196,54 @@ var microprofileConfigCallBack = (function() {
         return allMatch;      
     }
 
-    var __checkInjectionEditorContent = function(content) {
+    var __checkDefaultInjectionEditorContent = function(content) {
         var annotationIsThere = false;
         var editorContentBreakdown = __getInjectionConfigContent(content);
         if (editorContentBreakdown !== null) {
-            annotationIsThere = __isParamInAnnotation(editorContentBreakdown);
+            annotationIsThere = __isDefaultInjectParamInAnnotation(editorContentBreakdown);
         } 
         return annotationIsThere;
     };
 
-    var __listenToEditorForInjectConfig = function(editor) {
+    var __listenToEditorForInjectDefaultConfig = function(editor) {
         var __showWebBrowser = function() {
             var stepName = editor.getStepName();
             var content = contentManager.getEditorContents(stepName);
-            if (__checkInjectionEditorContent(content)) {
+            if (__checkDefaultInjectionEditorContent(content)) {
                 editor.closeEditorErrorBox(stepName);
-                //contentManager.showBrowser(stepName, 0);
-                //contentManager.addRightSlideClassToBrowser(stepName, 0);
+                contentManager.showBrowser(stepName, 0);
+                contentManager.addRightSlideClassToBrowser(stepName, 0);
                 contentManager.markCurrentInstructionComplete(stepName);
                 //contentManager.updateWithNewInstructionNoMarkComplete(stepName);
             } else {
                 // display error and provide link to fix it
-                editor.createErrorLinkForCallBack(stepName, true, __addInjectConfigToEditor);
+                editor.createErrorLinkForCallBack(stepName, true, __addInjectDefaultConfigToEditor);
             }
         };
         editor.addSaveListener(__showWebBrowser);
     };
 
-    var __addConfigInjectButton = function(event) {
+    var __addInjectDefaultConfigButton = function(event) {
         if (event.type === "click" ||
         (event.type === "keypress" && (event.which === 13 || event.which === 32))) {
             // Click or 'Enter' or 'Space' key event...
-            __addInjectConfigToEditor();
+            __addInjectDefaultConfigToEditor();
         }        
     };
 
-    var __addInjectConfigToEditor = function(stepName) {      
-        var injectConfig = "    @Inject @ConfigProperty(name=\"download-url\", defaultValue=\"ftp://music.com/us/download\")";
+    var __addInjectDefaultConfigToEditor = function(stepName) {      
+        var injectConfig = "    @Inject @ConfigProperty(name=\"download-url\", \n" +
+                           "                            defaultValue=\"ftp://music.com/us/download\")";
```

Comment:
I've changed the default to us-east. Please use ftp://music.com/us-east/download.